Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -974,19 +974,22 @@ void RenderObject::propagateRepaintToParentWithOutlineAutoIfNeeded(const RenderL
     ASSERT_NOT_REACHED();
 }
 
-void RenderObject::repaintUsingContainer(const RenderLayerModelObject* repaintContainer, const LayoutRect& r, bool shouldClipToLayer) const
+void RenderObject::repaintUsingContainer(SingleThreadWeakPtr<const RenderLayerModelObject> repaintContainer, const LayoutRect& r, bool shouldClipToLayer) const
```

Comment:
SingleThreadWeakPtr<const RenderLayerModelObject>&&

Reply:
This can't be done easily because `repaintUsingContainer()` is called multiple times from `repaintAfterLayoutIfNeeded()`.

Reply:
It can, just pass `WeakPtr { repaintContainer }` at call sites that can't WTFMove()..

Reply:
Still missing &&

Reply:
Fixed.I